CMBLensingSPT3GInterface/src/CMBLensingSPT3GInterface.jl

```diff
@@ -0,0 +1,223 @@
+module CMBLensingSPT3GInterface
+
+export Parent, Frame, MapSpectraTEB
+export @py_str
+
+
+using PyCall
+import PyCall: PyObject
+using CMBLensing
+using CMBLensing: unfold, FlatIEB, FlatIQU
+
+function __init__()
+    py"""
+    import numpy as np
+    from spt3g.core import G3Units
+    from spt3g.maps import MapProjection, FlatSkyMap, MapPolConv
+    from spt3g.mapspectra.map_spectrum_classes import MapSpectrum2D, MapSpectrum1D
+    from spt3g.mapspectra.basicmaputils import get_fft_scale_fac, map_to_ft
+    from spt3g.lensing.map_spec_utils import MapSpectraTEB
+    """
+
+    pytype_mapping(py"FlatSkyMap", FlatMap)
+    pytype_mapping(py"MapSpectrum2D", FlatFourier)
+    pytype_mapping(py"MapSpectraTEB ", FlatIEBFourier)
+    pytype_mapping(py"dict", FieldTuple)
+end
+
+Base.getindex(f::FlatField, s::String) = f[s == "T" ? :I : Symbol(s)]
```

Comment:
This is very convenient, but maybe a bit invasive...

Reply:
I think this is OK for now, it at least leaves a clear rule, its `:I,:E,:B`  on the Julia side and `"T", "E", "B"` on the Python side.

Fwiw my choice on the Julia side is because not all S0 fields are temperature, so it felt really wierd writing things like `ϕ.Tx`, so switched to slightly more generic `I` for intensity, like the intensity of the grav lensing potential or the intensity oft the deflection field somehow feels less obvoiusly wrong that the temperature of the grav lensing potential / deflection.

Reply:
Funny, I always thought it was to avoid confusion with Julia's `f(x::T) where T` convention